src/library/Uno.Material/Styles/Controls/Flyout.xaml

```diff
@@ -0,0 +1,528 @@
+									   HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+									   VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+									   Grid.Column="1" />
+
+							<TextBlock x:Name="KeyboardAcceleratorTextBlock"
+									   Text="{TemplateBinding KeyboardAcceleratorTextOverride}"
+									   Style="{StaticResource MaterialBody2}"
+									   Visibility="{Binding KeyboardAcceleratorTextOverride, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource MaterialEmptyOrNullToCollapsed}, FallbackValue=Collapsed, TargetNullValue=Collapsed}"
+									   Foreground="{ThemeResource ToggleMenuFlyoutItemKeyboardAcceleratorTextForeground}"
+									   VerticalAlignment="Center"
+									   TextAlignment="Right"
+									   AutomationProperties.AccessibilityView="Raw"
+									   Margin="16,0"
+									   Grid.Column="3" />
+						</Grid>
+						
+						<!-- Overlay -->
+						<Border x:Name="CommonStatesOverlay"
+								Background="Transparent" />
+					</Grid>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<!-- Material MenuFlyoutSubItem Style -->
+	<Style x:Key="MaterialMenuFlyoutSubItemStyle"
+		   TargetType="MenuFlyoutSubItem">
+
+		<Setter Property="Foreground"
+				Value="{StaticResource MaterialOnBackgroundBrush}" />
+		<Setter Property="Background"
+				Value="{StaticResource MaterialBackgroundBrush}" />
+		<Setter Property="Height"
+				Value="48" />
+
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="MenuFlyoutSubItem">
+
+					<Grid x:Name="LayoutRoot"
+						  Background="{TemplateBinding Background}"
+						  BorderBrush="{TemplateBinding BorderBrush}"
+						  BorderThickness="{TemplateBinding BorderThickness}"
+						  CornerRadius="{TemplateBinding CornerRadius}">
+
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualState x:Name="Normal" />
+
+								<VisualState x:Name="PointerOver">
+									<VisualState.Setters>
+										<Setter Target="LayoutRoot.(RevealBrush.State)"
+												Value="PointerOver" />
+										<Setter Target="CommonStatesOverlay.Background"
+												Value="{StaticResource MaterialOnSurfaceHoverBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="Pressed">
+									<VisualState.Setters>
+										<Setter Target="LayoutRoot.(RevealBrush.State)"
+												Value="Pressed" />
+										<Setter Target="CommonStatesOverlay.Background"
+												Value="{StaticResource MaterialOnSurfacePressedBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="SubMenuOpened">
+									<VisualState.Setters>
+										<Setter Target="LayoutRoot.(RevealBrush.State)"
+												Value="Pressed" />
+										<Setter Target="CommonStatesOverlay.Background"
+												Value="{StaticResource MaterialOnSurfacePressedBrush}" />
```

Comment:
It's a bit odd that PressedBrush is used for a Opened visual state. Semantically at least. But It might be the best solution I didn't look up the values. There's a *SelectedBrush also? is it similar?